apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/metrics/TenantRequestMetricsTagProvider.java

```diff
@@ -0,0 +1,75 @@
+package io.stargate.sgv2.api.common.metrics;
+
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import io.quarkus.micrometer.runtime.HttpServerMetricsTagsContributor;
+import io.stargate.sgv2.api.common.StargateRequestInfo;
+import io.stargate.sgv2.api.common.config.MetricsConfig;
+import io.vertx.core.http.HttpServerRequest;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import java.util.regex.Pattern;
+
+@ApplicationScoped
+public class TenantRequestMetricsTagProvider implements HttpServerMetricsTagsContributor {
+
+  // split pattern for the user agent, extract only first part of the agent
+  private static final Pattern USER_AGENT_SPLIT = Pattern.compile("[\\s/]");
```

Comment:
So it's either by white-space or `/`? (probably copied from another place this was done already?)

Reply:
Yes., It's copied from TenantRequestMetricsFilter